src/test/scala/chiselTests/ChiselSpec.scala

```diff
@@ -93,7 +93,7 @@ trait ChiselRunners extends Assertions with BackendCompilationUtilities {
   def compile(t: => RawModule): String = {
     (new ChiselStage)
       .execute(
-        Array("--target-dir", createTestDirectory(this.getClass.getSimpleName).toString),
+        Array("--target-dir", createTestDirectory(this.getClass.getSimpleName).toString, "--full-stacktrace"),
```

Comment:
Not sure if you meant to leave this, I think it's fine since this is for testing but thought I'd call it out.

Reply:
nope will roll this back